man/usr/share/man/man1m/piadm.1m.md

```diff
@@ -293,3 +341,10 @@ The following exit values are returned:
     if it's `zones`, a machine can still be booted with a USB stick, CD-ROM,
     or other method of booting SmartOS.  A bootable pool can then be
     repaired using piadm(1M) from the USB stick or CD-ROM.
+
+    Boot image customization is done at install time, other boot images
+    already present will not be updated. Older versions of piadm can still
+    be used to switch to and from boot images that have been customized but
+    won't be able to customize images. Because symlinks are used, changes
+    to common/loader.conf.local and common/loader.rc.local will be
+    be applied to all boot images that have been customized.
```

Comment:
If the paragraph change I suggested for hunk 1 of this file, I don't think this paragraph will be necessary.